### PR TITLE
Bug/sc 32750/a new object that was created is not bundled

### PR DESCRIPTION
**Pull request by Cwagne17**
This removes duplicate bundling in the exit builder process since README will trigger a bundle.

### CODE REVIEW

File: src/app/onion/learning-object-builder/learning-object-builder.component.ts [hunk 1]
```diff
@@ -119,7 +120,8 @@ export class LearningObjectBuilderComponent implements OnInit, OnDestroy {
     public noteService: ToastrOvenService,
     private authService: AuthService,
     private learningObjectService: LearningObjectService,
-    private history: HistoryService
+    private history: HistoryService,
+    private bundlingService: BundlingService,
```
Comment (Greek) [inline src/app/onion/learning-object-builder/learning-object-builder.component.ts:124]: is this used/needed?

File: src/app/onion/learning-object-builder/builder-store.service.ts [hunk 2]
```diff
@@ -475,6 +475,9 @@ export class BuilderStore {
       );
     }
   }
+
+
```
Comment (johngonz0) [inline src/app/onion/learning-object-builder/builder-store.service.ts:479]:
```suggestion
```

File: src/app/onion/learning-object-builder/builder-store.service.ts [hunk 3]
```diff
@@ -475,6 +475,9 @@ export class BuilderStore {
...
+
```
Comment (johngonz0) [inline src/app/onion/learning-object-builder/builder-store.service.ts:480]:
```suggestion

```

File: src/app/onion/learning-object-builder/learning-object-builder.component.ts [hunk 4]
```diff
@@ -132,7 +134,6 @@ export class LearningObjectBuilderComponent implements OnInit, OnDestroy {
         const cuid = routeParams.get('cuid');
         const version: number = +routeParams.get('version');
         const revision = this.route.snapshot.queryParamMap.get('revisionId');
-        const authorUsername = this.route.snapshot.queryParamMap.get('author');
```
Comment (johngonz0) [inline src/app/onion/learning-object-builder/learning-object-builder.component.ts:135]: Just curious as to why this is removed

**Review by johngonz0 — COMMENTED** [review]
Just a question

File: src/app/onion/learning-object-builder/builder-store.service.ts (hunk 3 above)
Comment (johngonz0) [reply src/app/onion/learning-object-builder/builder-store.service.ts:480]: i dont know why it only removed one space but i meant to remove 2

File: src/app/onion/learning-object-builder/learning-object-builder.component.ts (hunk 4 above)
Comment (Cwagne17) [reply src/app/onion/learning-object-builder/learning-object-builder.component.ts:135]: It was unused..

**Comment by Cwagne17** [comment]
A lot of the different work I was doing got merged down into this branch.

**Review by johngonz0 — APPROVED** [review]

**Pull request merged**